### PR TITLE
CVS-60551 Add OCR postprocessing to camera frames processing example

### PR DESCRIPTION
**Pull request by dkalinowski**
https://jira.devtools.intel.com/browse/CVS-60551

### CODE REVIEW

**Comment by dtrawins** [comment]
can we add also a short description to camera_example.md?

**Comment by dkalinowski** [comment]
> can we add also a short description to camera_example.md?

done

File: docs/camera_example.md [hunk 1]
```diff
@@ -63,3 +63,5 @@ python3 camera_client.py --grpc_address [hostname] --grpc_port 9001 --video_sour
 ```
 
 > **NOTE**: Video source is cropped to 704x704 resolution to match model input size.
+
+You can also specify `--use_case ocr` parameter to interpret outputs as OCR pipeline results. In this case the script will visualise bounding boxes together with recognized text words. To prepare model server configuration, follow this example: [link](east_ocr.md).
```
Comment (dtrawins) [inline docs/camera_example.md:67]:
```suggestion
You can also specify `--use_case ocr` parameter to interpret outputs as OCR pipeline results. In this case the script will visualise bounding boxes together with recognized text words. To prepare model server configuration, follow this [OCR pipeline example with EAST model](east_ocr.md).
```

File: docs/east_ocr.md [hunk 2]
```diff
@@ -224,3 +224,6 @@ The custom node generates the following text images retrieved from the original
 
 ## Accurracy
 Please note that it is possible to swap the models included in DAG with your own to adjust pipeline accuracy for various scenarios and datasets.
+
+## Images from camera
+You can also use examplary camera client with model server configured in this page. Refer to: [link](camera_example.md).
```
Comment (dtrawins) [inline docs/east_ocr.md:229]:
```suggestion
You can also use exemplary camera client with model server configured in this page. Refer to: [camera client example](camera_example.md).
```

**Review by rasapala — APPROVED** [review]
LGTM

**Comment by dkalinowski** [comment]
OCR is dropped due to model performance issues. Merge request is on hold.

**Comment by mkuczyns11** [comment]
This was resolved with the usage of horizontal text detection model and pipeline created here - https://github.com/openvinotoolkit/model_server/pull/1027.

**Pull request closed without merging**